ledfx/api/get_image.py

```diff
@@ -0,0 +1,59 @@
+import io
+import logging
+from json import JSONDecodeError
+
+import pybase64
+from aiohttp import web
+
+from ledfx.api import RestEndpoint
+from ledfx.utils import open_image
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class GetImageEndpoint(RestEndpoint):
+    ENDPOINT_PATH = "/api/get_image"
+
+    async def get(self, request: web.Request) -> web.Response:
+        """Open image and return as base64 encoded string
+
+        Args:
+            request (web.Request): The request object containing the `path_url` to open.
+
+        Returns:
+            web.Response: The HTTP response object containing the base64 encoded image.
+        """
+        try:
+            data = await request.json()
+        except JSONDecodeError:
+            return await self.json_decode_error()
+
+        path_url = data.get("path_url")
+
+        if path_url is None:
+            return await self.invalid_request(
+                'Required attribute "path_url" was not provided'
+            )
+
+        _LOGGER.info(f"GetImageEndpoint from {path_url}")
```

Comment:
Don't need this, all logging of API is done at the __init__ level - automagically will capture this.